substrafl/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.40.0"
+__version__ = "0.41.0"
```

Comment:
Don't you also need to update the Substra version in `setup.py`?

Reply:
The version was updated in the Pydantic PR, that needed this version to be functional.

Reply:
So main is already aligned with the last substra version